### ISSUE TITLE
Timeout

### ISSUE THREAD

**Issue by aquaglia** [inspire-eu-validation/download-service#47]
Please ensure a maximum timeout is specified in each ATS requesting probes of links.

**Comment by thijsbrentjens** [comment]
How should we determine the timeout value? Is this an implementation choice maybe? Or should we determine this for all ATS in general, based on the QoS requirements if applicable? The latter means we are writing ATS for QoS as well.

**Comment by aquaglia** [comment]
Sorry, I thought I had answered this already.
As I wrote in a similar issue, not including a timeout value would be a serious conceptual error.
QoS values are an excellent starting point and constitute a lower bound.
As we know, they do not take into account latency between the service and the consumer of the service.
We can start from the values specified in the QoS and agree on a multiplying factor.
This also useful when testing links to a generic HTTP resource like a web page. We can assimilate such a request similar to a Discover Metadata Request, since it is, in some sense, metadata about the described resource.
We can write a table where we categorize each HTTP request in a similar way.
